source/Login/LoginJS.js

```diff
@@ -106,7 +106,8 @@ function handleSignup(newUsername, newPassword) {
  */
 function verifyValidInputs(newUsername, newPassword) {
     //prohibit empty username
-    if (newUsername == '') {
+    console.log('here');
```

Comment:
this looks like a debugging log. If so perhaps you want to remove it before merging. Otherwise ignore this comment